project_resource_calendar/i18n/fr.po

```diff
@@ -939,48 +972,3 @@ msgstr "Rapport d'événement hebdomadaire"
 #: model:ir.model,name:project_resource_calendar.model_weekly_report_wizard
 msgid "weekly.report.wizard"
 msgstr "weekly.report.wizard"
-
```

Comment:
Dont delete the following translations. The following translations are added manually. that's why Poedit delete them each time but we must keep them.